.github/workflows/push.yml

```diff
@@ -0,0 +1,29 @@
+on: [push, pull_request]
+
+name: CI / GitHub Actions
+
+jobs:
+  installDependencies:
+    name: NodeJS
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
```

Comment:
same thing:
```
action "Only publish master branch" {
  uses = "BinaryMuse/tip-of-branch@master"
  needs = ["Run tests"]
  args = "master"
  secrets = ["GITHUB_TOKEN"]
}
```
is missing it seems

Reply:
Resolved for same reason as https://github.com/electron/electron-api-historian/pull/9#discussion_r314178635